.changeset/brave-colts-cover.md

```diff
@@ -0,0 +1,25 @@
+---
+"astro": minor
+---
+
+It's now possible the render components in isolation with the new Astro Container APIs:
+
+If you use `vitest`, you can quickly write a test like this:
+
+```js
+import { experimental_AstroContainer as AstroContainer } from 'astro/container';
+import { expect, test } from 'vitest';
+import Card from '../src/components/Card.astro';
+
+test('Card with slots', async () => {
+	const container = await AstroContainer.create();
+	const result = await container.renderToString(Card, {
+		slots: {
+			default: 'Card content',
+		},
+	});
+
+	expect(result).toContain('This is a card');
+	expect(result).toContain('Card content');
+});
+```
```

Comment:
``````suggestion
```

For a complete reference, see the [Container API docs](/en/reference/container-reference/).

For a feature overview, and to give feedback on this experimental API, see the [Container API roadmap discussion](https://github.com/withastro/roadmap/pull/916).
``````
I would at least include links to the docs, but I think it would also be good to link to the discussion!